examples/with-tailwind/apps/web/next.config.ts

```diff
@@ -2,4 +2,10 @@
 module.exports = {
   reactStrictMode: true,
   transpilePackages: ["@repo/ui"],
+  typescript: {
+    ignoreBuildErrors: true,
+  },
+  eslint: {
+    ignoreDuringBuilds: true,
+  },
```

Comment:
Needed?

Reply:
Yes, these don't need to be in the blocking path of a build since we're Turborepo and we do `lint` and `check-types` tasks instead.